crates/astria-cli/tests/create_account.rs

```diff
@@ -0,0 +1,75 @@
+use std::{
+    fs,
+    path::Path,
+};
+
+use assert_cmd::Command;
+use astria_eyre::{
+    eyre::WrapErr,
+    Result,
+};
+use predicates::prelude::predicate;
+
+struct Resources {
+    input: String,
+    expected_verbose_display: String,
+}
+
+impl Resources {
+    /// Reads the contents of the files in the `tests/resources/parse_blob/<test_case>` folder to
+    /// the respective fields of `Self`.
+    fn new(test_case: &str) -> Result<Self> {
+        let dir = Path::new(env!("CARGO_MANIFEST_DIR"))
+            .join("tests")
+            .join("resources")
+            .join(test_case);
+        let read_file = |filename: &str| -> Result<String> {
+            let path = dir.join(filename);
+            fs::read_to_string(&path).wrap_err(format!("failed to read {}", path.display()))
+        };
+        Ok(Resources {
+            input: read_file("input.txt")?,
+            expected_verbose_display: read_file("expected_verbose_output.txt")?,
+        })
+    }
+
+    #[track_caller]
+    fn check_reconstruct_account(self) -> Result<()> {
+        let mut cmd = new_create_account_command()?;
+        cmd.arg("--from-mnemonic")
+            .arg(&self.input)
+            .assert()
+            .success()
+            .stdout(predicate::eq(self.expected_verbose_display.clone()));
+
+        Ok(())
+    }
+}
+
+fn new_create_account_command() -> Result<Command> {
+    // astria-cli sequencer account create command
+    let mut cmd = Command::cargo_bin(env!("CARGO_PKG_NAME"))?;
+    cmd.arg("sequencer").arg("account").arg("create");
+    Ok(cmd)
+}
+
+#[test]
+fn should_reconstruct_account() -> Result<(), Box<dyn std::error::Error>> {
+    let resources: Resources = Resources::new("create_account")?;
+    resources.check_reconstruct_account()?;
+    Ok(())
+}
+
+#[test]
+#[track_caller]
```

Comment:
This is only worth adding to functions called from within the test, not the test itself.  I'd move this to `fn new_create_account_command()`.